app/models/risk_model.py

```diff
@@ -18,5 +17,34 @@ class RiskModel(db.Model):
 
     id = Column(Integer, primary_key=True)
     title = Column(String, nullable=False, unique=True)
-    description = Column(String, nullable=False)
     text = Column(String, nullable=False)
+
+    CASES = {
+        "NOT_SPECIFIED": {"live_nearby_river": False, "live_nearby_mountain": False},
+        "EARTH_SLIDING": {"live_nearby_river": False, "live_nearby_mountain": True},
+        "FLOOD": {"live_nearby_river": True, "live_nearby_mountain": False},
+        "FLOOD_AND_SLIDING": {"live_nearby_river": True, "live_nearby_mountain": True},
+    }
+
+    # TODO: find a better way to display the texts
+    VALUES = {
+        "NOT_SPECIFIED": """Atenção! A sua região apresenta previsão para chuvas fortes (ou violentas <depende da milimetragem>). 
```

Comment:
Messages must contain a maximum of 160 characters (SMS limit). We need to change it in the future.